modules/pcd/src/lib/parse-pcd.ts

```diff
@@ -336,9 +333,9 @@ function parsePCDBinary(pcdHeader: PCDHeader, data: ArrayBufferLike): HeaderAttr
     }
 
     if (offset.rgb !== undefined) {
-      color.push(dataview.getUint8(row + offset.rgb + 0));
-      color.push(dataview.getUint8(row + offset.rgb + 1));
-      color.push(dataview.getUint8(row + offset.rgb + 2));
+      color.push(dataview.getUint8(row + offset.rgb + 2) / 255.0);
```

Comment:
Agree that using normalized colors is better. However this is a breaking change and should probably be mentioned in `docs/upgrade-guide.md`. 

I am on the fence if it is worth it, but we could avoid breaking if we offered a `pcd.normalizeColors` loader option on the PCDLoader.

I have some concerns that we have code that creates a typed array from this array and if that typed array is Uint8Array it might fail (all values will be 0 or 1). Not sure if it is easy for you to find, I will try to dig more into this when I find some time.

Reply:
> Not sure what you mean by the last section about the Uint8Array. I think this is not applicable when we introduce a new array, right?

There is code in loaders.gl that can convert parsed data to e.g. binary representations, in which case it might copy this JS array into a Uint8Array.

I'd have to dig into exactly what is implemented for meshes in terms of conversion right now to verify if it is an issue at this time.